src/packages/utils/src/utils/request-coordinator.ts

```diff
@@ -97,6 +124,7 @@ export class RequestCoordinator {
           reject(e);
         }
       };
+      executor.reject = reject;
```

Comment:
I'm not fond of extending the function itself with an additional property. What are your thoughts about doing something like:

```
this.pending.push({ executor, reject });
```

instead?

Reply:
Yep, I like that.